### PR TITLE
Most popular by industry

### PR DESCRIPTION
**Pull request by adgad**
/cc @ironsidevsquincy @matthew-andrews 

* Add dropdown to select most popular by industry
* Refactors the way Homepage is rendered:  (Previously: Layout -> Section -> Cards, Now:  Section -> Cards ). This allows us to only render certain sections client side (and therefore not have to include the entire JSON of the page).
* Reduce DOM size by only rendering dynamic sections (or if the layout tool flag is on) as "react" divs.

TODO:
- [x] Create feature flag
- [x] Put behind feature flag
- [x] Add other industries
- [x] Add "all" to dropdown somehow
- [x] Fix "Warning: render(...): Replacing React-rendered children with a new root component..."
- [x] Styling
- [x] IE9/CORS
- [x] Core experience (maybe just hide)
- [x] Persist choice to localStorage

### CODE REVIEW

File: components/section/browser.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import SectionNode from './node';
+import nJsonpFetch from 'n-jsonp-fetch';
+import Superstore from 'superstore';
+const fetchres = require('fetchres');
```
Comment (ironsidevsquincy) [inline components/section/browser.js]: can this be `import`ed too?

File: components/section/browser.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+	constructor(props) {
+		super(props);
+		this.defaultSourceStore = new Superstore('local', 'next-frontpage');
+	}
+
+ 	componentWillMount() {
+ 		//If there is a source stored in localStorage, fetch that
+ 		if(this.props.dynamicContent && this.props.dynamicContent.rememberSource) {
+ 			let key = this.props.dynamicContent.rememberSource;
```
Comment (ironsidevsquincy) [inline components/section/browser.js]: `const`?

File: components/section/browser.js (hunk 1 above)
Comment (adgad) [inline components/section/browser.js]: import seems to break fetchres - haven't dug into why though

Comment (adgad) [inline components/section/browser.js]: actually hold that thought...

File: components/section/main.scss [hunk 3]
```diff
@@ -83,6 +84,18 @@
 	flex-direction: column;
 }
 
+.section-sources__label {
+	font-family: $n-type-sans-serif;
```
Comment (ironsidevsquincy) [inline components/section/main.scss]: `o-type-...` instead

**Pull request merged**